config/feature_toggles.yml

```diff
@@ -0,0 +1,3 @@
+---
+edit_registration:
+  active: <%= ENV["FEATURE_TOGGLE_EDIT_REGISTRATION"] || "true" %>
```

Comment:
I'd be tempted to set this to FALSE by default as then we don't have to make any env var changes on production.

Reply:
The FeatureToggle class will return false if the env variable is not set, so we just need to remove this `|| true`